mujinvisioncontrollerclient/visioncontrollerclient.py

```diff
@@ -399,110 +422,241 @@ def StartVisualizePointCloudTask(self, vminitparams, locationName=None, sensorSe
-        if sensorSelectionInfo:
+        if sensorSelectionInfo is not None:
             command['sensorSelectionInfo'] = sensorSelectionInfo
-        if metadataOnly:
-            command['metadataOnly'] = metadataOnly
-        if imageTypes:
+        if imageTypes is not None:
             command['imageTypes'] = imageTypes
-        if limit:
+        if limit is not None:
             command['limit'] = limit
         return self._ExecuteCommand(command, timeout=timeout, recvjson=False, blockwait=blockwait)
 
     def WaitForGetLatestDetectionResultImages(self, timeout=2.0):
+        """Waits for response to GetLatestDetectionResultImages command
+
+        Args:
+            timeout (float, optional): Time in seconds after which the command is assumed to have failed. (Default: 2.0)
+        """
         return self._WaitForResponse(recvjson=False, timeout=timeout)
-    
+
+
     def GetDetectionHistory(self, timestamp, timeout=2.0):
         # type: (int, float) -> Any
         """Gets detection result with given timestamp (sensor time)
-        
+
         Args:
-            timestamp (int): unix timestamp in milliseconds
+            timestamp (int): Unix timestamp in milliseconds
+            timeout (float, optional): Time in seconds after which the command is assumed to have failed. (Default: 2.0)
+
+        Returns:
+            str: Binary blob of detection data
         """
-        log.verbose("Getting detection result at %r ...", timestamp)
         command = {
             'command': 'GetDetectionHistory',
-            'timestamp': timestamp
+            'timestamp': timestamp,
         }
         return self._ExecuteCommand(command, timeout=timeout, recvjson=False)
     
     def GetVisionStatistics(self, taskId=None, cycleIndex=None, taskType=None, timeout=2.0):
-        """Gets the latest vision stats
+        """Gets the latest vision stats.
+
+        Args:
+            taskId (str, optional): If specified, the taskId.
+            cycleIndex (str, optional): Unique cycle index string for tracking, backing up, and differentiating cycles.
+            taskType (str, optional): The task type.
+            timeout (float, optional): Time in seconds after which the command is assumed to have failed. (Default: 2.0)
+
+        Returns:
+            dict: a list of all currently active vision task statistics. Each task statistics have the following structure
+            A dictionary with the structure:
+
+            visionStatistics (dict): A dictionary with the structure:
+
+                cycleIndex (str): Unique cycle index string for tracking, backing up, and differentiating cycles.
+                taskId (str): The taskId.
+                taskType (str): The task type.
+                taskStartTimeMS (int)
+                totalDetectionTimeMS (int)
+                totalDetectionCount (int)
+                totalGetImagesCount (int)
+                targetURIs (int)
+                detectionHistory (list)
```

Comment:
where did you get this structure from?

Reply:
Same as above, Aditi read visionmanager. I didn't check manually because I expect these types to be checked automatically in tests at some point.